README.md

```diff
@@ -1,29 +1,54 @@
 # JIRA Chrome extension
-A chrome extension that queries a public JIRA API (https://jira.secondlife.com).  This is a fully functional chrome extension with a bug, some display issues, and generally some bad practices.  
-
-**Your task is to make it production ready, make it easy for someone else to consume, and generally improve upon the code how you see fit.**
+A chrome extension that queries a public JIRA API (https://jira.secondlife.com). 
 
 ## Files:
-* main.js - the :meat_on_bone: of the code that performs querys and displays results
-* main.html - The UI for the request results
-* manifest.json - contains metadata for the project and defines access permissions
-* options.js - contains logic for saving/retrieving user values
-* options.html - display for the user values
-* json_results/results.json - a sample of the json values returned from a JIRA ticket(s) query
+* **scripts/main.js** - Contians the main logic for the extension
+* **scripts/options.js** - Contains the logic for saving and defaulting prefs
+* **pages/main.html** - The UI for the main page.
+* **pages/options.html** - display for the user values
+* **js/alerts.js** - Common alert functionality
+* **js/fields.js** - Common field functionality
+* **js/tabs.js** - Common tabs functionality
+* **js/http.js** - Common http/get functionality
+* **js/functions.js** - Common used functions
+* **manifest.json** - contains metadata for the project and defines access permissions
```

Comment:
#nit Inconsistent uppercase/lowercase on a few of the file descriptions' first word